README.md

```diff
@@ -2,11 +2,9 @@
 
 # Clarity Design System
 
-[![Build Status](https://travis-ci.com/vmware/clarity.svg?branch=master)](https://travis-ci.com/vmware/clarity)
-
-[![Build Status](https://saucelabs.com/browser-matrix/claritydesignsystem.svg)](https://saucelabs.com/beta/builds/b16110e384ce459ab68f10da6e38a285)
-
-[![npm version](https://badge.fury.io/js/%40clr%2Fcore.svg)](https://badge.fury.io/js/%40clr%2Fcore)
+| Build                                                                                                           | NPM                                                                                                  |
+| --------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------- |
+| [![Build Status](https://travis-ci.com/vmware/clarity.svg?branch=master)](https://travis-ci.com/vmware/clarity) | [![npm version](https://badge.fury.io/js/%40clr%2Fcore.svg)](https://badge.fury.io/js/%40clr%2Fcore) |
```

Comment:
I think that these build/package icons are not valid anymore.

* We don't use Travis CI for a long time so we need to update them with the Github Action more [here](https://docs.github.com/en/free-pro-team@latest/actions/managing-workflow-runs/adding-a-workflow-status-badge)
* NPM Version badge is targeting `clr/core` package that is not the only package that we have. So adding other packages will be a best


![npm version core](https://img.shields.io/npm/v/@cds/core/latest?label=%40cds%2Fcore&style=flat-square)
![npm version angular](https://img.shields.io/npm/v/@clr/angular/latest?label=%40clr%2Fangular&style=flat-square)
![npm version cds angular](https://img.shields.io/npm/v/@cds/angular/latest?label=%40cds%2Fangular&style=flat-square) - when released
![npm version ui](https://img.shields.io/npm/v/@clr/ui/latest?label=%40clr%2Fui&style=flat-square)
![npm version icons](https://img.shields.io/npm/v/@clr/icons/latest?label=%40clr%2Ficons&style=flat-square)

```
![npm version icons](https://img.shields.io/npm/v/@clr/icons/latest?label=%40clr%2Ficons&style=flat-square)
```